spec/support/step_nav_steps.rb

```diff
@@ -122,6 +122,19 @@ def given_a_step_by_step_has_been_updated_after_links_last_checked
     )
   end
 
+  def given_a_step_by_step_has_had_an_empty_step_added_after_links_last_checked
```

Comment:
Is there a way this could be shortened (it took me a while to process it)?

Although I'm struggling to think of anything much shorter 🤔: 
`given_a_step_by_step_has_an_empty_step_added_without_links_checked`
`given_a_step_by_step_has_an_empty_step_added_after_links_last_checked`